loader.js

```diff
@@ -749,8 +773,11 @@ var Module = null;
      return args;
    };
    
-   var build_vice_arguments = function (emulator_start, files, extra_args) {
+   var build_vice_arguments = function (emulator_start, files, fliplist, extra_args) {
        var args = emulator_start ? ["-autostart", "/emulator/" + emulator_start] : [];
+       if(fliplist[0] || fliplist[1] || fliplist[2] || fliplist[3]) {
+           args = args.concat(["-flipname", "/emulator/metadata_fliplist.vfl"]);
+       }
```

Comment:
This patch works, but needs a small fix. The `VICELoader` assumes that a file with the fliplist instructions exists and has a specific name, but only the `IALoader` creates that file; a user that doesn't want to use `IALoader` will have to do that themselves, as well as call the `fliplist` method on `VICELoader`.

To fix this, just move the code that creates the fliplist file down here into the `VICELoader`

Reply:
I'm confused, are you saying that the problem is for non-`IALoader` users who want to use `VICELoader` with fliplists, or all non-`IALoader` users who want to use `VICELoader`?

For the latter, maybe I should be checking if `fliplist` is defined in that snipit you hilighted, but I don't see any other problems? Although I haven't tested.

Reply:
The former. If I set up a page that uses VICELoader, and I call `VICELoader.fliplist([ ... ])`, it'll fail to work because it ignores what I pass in and assumes the file already exists.